src/js/components/Tab/index.d.ts

```diff
@@ -2,6 +2,8 @@ import * as React from 'react';
 import { Omit } from '../../utils';
 
 export interface TabProps {
+  a11yTitle?: string;
+  active?: string;
```

Comment:
As far as I am aware, there is no `active` prop on the Tab component. The solution for the `styled-components` TypeScript error should look something like [this](https://stackoverflow.com/questions/52404958/using-styled-components-with-typescript-prop-does-not-exist): 
```Typescript
  const ButtonTab = styled(Tab)<{active: boolean}>`
    ${(props) => css`
      background: ${props.active ? 'blue' : 'green'};
    `}
  `;
```
@jcfilben Let me know if this seems about right, I have very little experience with `styled-components`.

Reply:
That looks right to me, thanks for your help reviewing this @alexmirrington!